src/algebra/module/basic.lean

```diff
@@ -552,9 +552,13 @@ variables (M)
 
 lemma smul_right_injective [no_zero_smul_divisors R M] {c : R} (hc : c ≠ 0) :
   function.injective (λ (x : M), c • x) :=
```

Comment:
While we're here, `apply smul_right_injective` produces a cleaner goal if you write it as
```suggestion
  function.injective ((•) c : M → M) :=
```